### ISSUE TITLE
`:VimwikiShowVersion` (still) broken on dev?

### ISSUE THREAD

**Issue by beuerle** [vimwiki/vimwiki#654]
Not sure if this is related to #649, but currently I am not able to run `:VimwikiShowVersion`.

I am getting the following error message:

```vim
Error detected while processing function <SNR>49_get_version:
line    4:
E461: Illegal variable name: a:plugin_rev
line    5:
E461: Illegal variable name: a:plugin_branch
line    6:
E461: Illegal variable name: a:plugin_date
line    8:
E121: Undefined variable: a:plugin_branch
line    9:
E121: Undefined variable: a:plugin_rev
line   10:
E121: Undefined variable: a:plugin_date
```

I am sitting on the `dev` branch with the most recent commit e0fffb4.

**Comment by ranebrown** [comment]
@beuerle I'm not seeing that error running the `dev` branch. I suspect this could be related to this [vim patch](https://github.com/vim/vim/commit/31b816042fca879b11965ddd75287732563ba698). What version of Vim are you running?

**Comment by ranebrown** [comment]
On a side note `VimwikiShowVersion` does seem to be broken on master with this error:
```vim
Error detected while processing function <SNR>23_get_version:
line    2:
E806: using Float as a String
```

**Comment by beuerle** [comment]
> @beuerle I'm not seeing that error running the `dev` branch. I suspect this could be related to this [vim patch](https://github.com/vim/vim/commit/31b816042fca879b11965ddd75287732563ba698). What version of Vim are you running?
> 
> I'm running 8.1.577

```
VIM - Vi IMproved 8.1 (2018 May 18, compiled Mar 29 2019 20:08:59)
Included patches: 1-1073
```

**Comment by ranebrown** [comment]
Can you replace the function `s:get_version()` in `plugin/vimwiki.vim` with the below function and let me know if that works?
```vim
function! s:get_version()
  if s:plugin_vers != -1
    echo "Stable version: " . s:plugin_vers
  else
    let l:plugin_rev    = system("git --git-dir " . s:plugin_dir . "/.git rev-parse --short HEAD")
    let l:plugin_branch = system("git --git-dir " . s:plugin_dir . "/.git rev-parse --abbrev-ref HEAD")
    let l:plugin_date   = system("git --git-dir " . s:plugin_dir . "/.git show -s --format=%ci")
    if v:shell_error == 0
      echo "Branch: " . l:plugin_branch
      echo "Revision: " . l:plugin_rev
      echo "Date: " . l:plugin_date
    else
      echo "Unknown version"
    endif
  endif
endfunction
```

**Comment by ranebrown** [comment]
Actually just pull the latest `dev` branch and give that a try.

**Comment by ranebrown** [comment]
@Nudin How to you want to go about merging this fix to `master` as v2.4.1? I think we want to ensure the version command works since including that output is part of our issue template.

**Comment by beuerle** [comment]
> Actually just pull the latest `dev` branch and give that a try.

it works, thanks for the quick fix!
